test/kwtool_web/params/search_params_test.exs

```diff
@@ -0,0 +1,32 @@
+defmodule KwtoolWeb.SearchParamsTest do
```

Comment:
Let's move to the same folder structure as the module -> `KwtoolWeb.Api.V1.SearchParamsTest` 👍

Reply:
Updated in https://github.com/byhbt/kwtool/pull/49/commits/9eed60c516f4bc84c9d0a38b0303479f65f0354a